### PR TITLE
Add in the lab requested alignment patches.

### PR DESCRIPTION
**Pull request by jallisonciq**
Add in the lab requested alignment patches.

### CODE REVIEW

File: lib/nettle/pk.c [hunk 1]
```diff
@@ -1735,10 +1758,7 @@ _wrap_nettle_pk_verify(gnutls_pk_algorithm_t algo,
 			if (hash_len > vdata->size)
 				hash_len = vdata->size;
 
-			/* SHA-1 is allowed for SigVer in FIPS 140-3 in legacy
-			 * mode */
 			switch (DIG_TO_MAC(sign_params->dsa_dig)) {
-			case GNUTLS_MAC_SHA1:
```
Comment (jason-rodri) [inline lib/nettle/pk.c:1741]: are we removing SHA-1?

File: tests/fips-test.c [hunk 2]
```diff
@@ -594,15 +594,15 @@ void doit(void)
...
+	/* Verify a signature created with 2432-bit RSA and SHA-1; not approved */
 	FIPS_PUSH_CONTEXT();
 	ret = gnutls_pubkey_verify_data2(pubkey, GNUTLS_SIGN_RSA_SHA1,
 					 GNUTLS_VERIFY_ALLOW_SIGN_WITH_SHA1, &data,
 					 &rsa2342_sha1_sig);
 	if (ret < 0) {
 		fail("gnutls_pubkey_verify_data2 failed\n");
 	}
-	FIPS_POP_CONTEXT(APPROVED);
+	FIPS_POP_CONTEXT(NOT_APPROVED);
```
Comment (jason-rodri) [inline tests/fips-test.c:605]: Same as above are you removing sha1?

File: tests/fips-test.c [hunk 3]
```diff
@@ -706,15 +706,15 @@ void doit(void)
...
+	/* Verify a signature created with ECDSA and SHA-1; not approved */
 	FIPS_PUSH_CONTEXT();
 	ret = gnutls_pubkey_verify_data2(pubkey, GNUTLS_SIGN_ECDSA_SHA1,
 					 GNUTLS_VERIFY_ALLOW_SIGN_WITH_SHA1, &data,
 					 &ecc256_sha1_sig);
 	if (ret < 0) {
 		fail("gnutls_pubkey_verify_data2 failed\n");
 	}
-	FIPS_POP_CONTEXT(APPROVED);
+	FIPS_POP_CONTEXT(NOT_APPROVED);
```
Comment (jason-rodri) [inline tests/fips-test.c:717]: Are we removing sha1?

File: lib/nettle/pk.c (hunk 1 above)
Comment (jallisonciq) [reply lib/nettle/pk.c:1741]: This is SHA1 for public key signing, not SHA1 in general.

File: tests/fips-test.c (hunk 2 above)
Comment (jallisonciq) [reply tests/fips-test.c:605]: Again, this is SHA1 in a public key sign/verify operation, not in general.

File: tests/fips-test.c (hunk 3 above)
Comment (jallisonciq) [reply tests/fips-test.c:717]: GNUTLS_VERIFY_ALLOW_SIGN_WITH_SHA1 - public key sign operation only.

**Review by jason-rodri — APPROVED** [review]

**Pull request merged**